### PR TITLE
Header update

### PR DESCRIPTION
**Pull request by RCdeWit**
Add a blinking cursor to the header so that visitors immediately know we work with computers.

### CODE REVIEW

**Review by jellebouwman — CHANGES_REQUESTED** [review]
Could you supplement this PR with a screen recording of the newly implemented animation?

Ideally a 0.25x, 0,5x speed version are included as well, but I'll take what I can get.

**Comment by RCdeWit** [comment]
Please see here, @jellebouwman:

https://user-images.githubusercontent.com/7165065/223421921-cb4fb537-8f04-4c4e-9a40-af7661b60903.mov

File: style.css [hunk 1]
```diff
@@ -32,3 +32,18 @@ body {
...
+
+@keyframes blink-animation {
+    to {
+      visibility: hidden;
+    }
+  }
+@-webkit-keyframes blink-animation {
+to {
+    visibility: hidden;
+}
```
Comment (jellebouwman) [inline style.css]:
```suggestion
  to {
  visibility: hidden;
  }
```

**Review by jellebouwman — CHANGES_REQUESTED** [review]
I'm getting seasick as I'm looking at this. Please consider users that have set their OS's to signal [prefers-reduced-motion](https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-reduced-motion). Let's be as inclusive as possible.

Consider adding

```CSS
@media (prefers-reduced-motion) {
  .blink {
    animation: none;
    visibility: visible
  }
}
```

to the bottom of your `style.css` file.

**Comment by RCdeWit** [comment]
I have incorporated the change you suggested. Nonetheless, I don't think we should stall the merge based on edge cases like these. The change, as it stands, is a major improvement to the current header that immediately clarifies our purpose and mission.

The vast majority of our users will benefit from this change. We can (and should) iterate further in the next sprint.

**Comment by jellebouwman** [comment]
> I have incorporated the change you suggested. Nonetheless, I don't think we should stall the merge based on edge cases like these. The change, as it stands, is a major improvement to the current header that immediately clarifies our purpose and mission.
> 
> The vast majority of our users will benefit from this change. We can (and should) iterate further in the next sprint.

I'd appreciate some user research on _the vast majority_ of our users benefiting from this change - this feels unsupported. 

Agreed to move on for now. One last thing to address - @RCdeWit could you please consider signing your commits? More info [here](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits). Happy to jump on a call if you need some help with that.

**Comment by jellebouwman** [comment]
Ah, I see our head of security has merged this PR with gaping holes in our security.

**Pull request merged**